portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase

```diff
@@ -0,0 +1,120 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Verify that a form can be submited with fields in two different languages,"
+	@priority = 5
+	test CheckIsPossibleSubmitFormWithTwoDifferentLanguages {
+		task ("Add a form with a text field defined as Full Name") {
+			FormsAdmin.addForm();
+
+			Form.editName(formName = "New Forms");
+
+			Form.gotoAddField(
+				fieldPositionNumber = 0,
+				fieldType = "Text");
+
+			Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+			Form.closeSidebar();
+		}
+
+		task ("Add a numerical field defined as Age") {
+			Form.gotoAddField(
+				fieldPositionNumber = 1,
+				fieldType = "Numeric");
+
+			Form.editFieldLabelMetal(fieldValue = "Age");
+		}
+
+		task ("Change the language to portuguese, add* new values and submit again") {
+			Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Full Name",
+				label = "Nome Completo");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Age",
+				label = "Idade");
+		}
+
+		task ("We publish the form, save the form, and navigate to the published form") {
+			Form.publishForm();
+
+			Form.save();
+
+			FormsAdminNavigator.gotoPublishedForm();
+		}
+
+		task ("Aentering values into the form and sending the entered values") {
+			FormViewBuilder.editText(
+				fieldName = "Full Name",
+				fieldValue = "Henrique");
+
+			FormViewBuilder.editText(
+				fieldName = "Age",
+				fieldValue = 23);
+
+			FormPortlet.submit();
+
+			Form.submitAgain();
+		}
+
+		task ("Changing the language to portuguese and inserting new values and sending") {
+			FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
+			FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "português-Brasil");
+
+			FormViewBuilder.editText(
+				fieldName = "Nome Completo",
+				fieldValue = "Jose Henrique Santos de lima");
+
+			FormViewBuilder.editText(
+				fieldName = "Idade",
+				fieldValue = 23);
+
+			FormPortlet.submit();
+
+			Form.submitAgain();
+		}
+
+		task ("Navigate to portal and go to forms") {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Forms");
+		}
+
+		task ("Check if the entries are present with the correct values") {
+			FormsAdminNavigator.gotoViewEntries();
+
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Henrique",
+				key_rowEntry = 23);
+
+			LexiconEntry.viewEntryContent(
+				key_rowAny = "Jose Henrique Santos de lima",
+				key_rowEntry = 23);
+		}
+	}
+
```

Comment:
- Please check : https://github.com/lucasperj/liferay-portal/pull/372#issuecomment-1706815755

Finally, what we talked about on Friday, the ideal here would be for you to either edit the existing commit to change the wrong message OR reset it to the upstream hash after the rebase and commit again, but not put the other commit on top with just the changes and practically the same commit message, in the commit history the commit with the wrong message still appears and makes it more confusing when reviewing, please fix that.